poll/public/handlebars/poll_results.handlebars

```diff
@@ -0,0 +1,43 @@
+<script id="poll-results-template" type="text/html">
+    <h3 class="poll-header">{{display_name}}</h3>
+    <div class="poll-question-container">{{{question}}}</div>
+    <ul class="poll-answers-results poll-results">
+    {{#each tally}}
+        <li class="poll-result">
+            <div class="poll-result-input-container">
+              <input id="answer-{{key}}" type="radio" disabled {{#if choice}}checked="True"{{/if}} />
+            </div>
+            {{#if any_img}}
+                <div class="poll-image result-image">
+                    <label for="answer-{{key}}" class="poll-image-label">
+                        {{#if img}}
+                          <img src="{{img}}" />
```

Comment:
@sarina That would be possible, but I feel it would be a bit weird to force people who change something unrelated to add the alt descriptions.  Moreover, this XBlock is also used outside of edx.org, in contexts that have different accessibility requirements.  And finally, as stated before, I don't think people will enter anything useful just because they are forced to enter _something_.